CHANGELOG.md

```diff
@@ -34,6 +34,7 @@ This feature release requires a **minimum Go version of v1.19.12 or higher to su
   - feat: Lotus Gateway: add MpoolPending, ChainGetBlock and MinerGetBaseInfo ([filecoin-project/lotus#10929](https://github.com/filecoin-project/lotus/pull/10929))
 
 ## Improvements
+  - chore: Auto remove local chain data when importing chain file or snapshot ([filecoin-project/lotus#11277](https://github.com/filecoin-project/lotus/pull/11277))
```

Comment:
Thanks for including this! It needs to be moved up to the `UNRELEASED` section, though

Reply:
Thanks. Fixed